examples/detail/app.js

```diff
@@ -0,0 +1,153 @@
+/*global FastList*/
+'use strict';
+
+var debug = 0 ? (...args) => console.log('[DataSource]', ...args) : ()=>{};
+
+var content = document.getElementById('content');
+var data = createDummyData(1000);
+var itemHeight = 88;
+
+var list = new FastList({
+  container: content,
+
+  createItem: function() {
+    var section = document.createElement('li');
+    section.innerHTML = '<h3> </h3><p> </p><div><img /></div>';
+    return section;
+  },
+
+  createSection: function() {
+    var section = document.createElement('section');
+    section.innerHTML = '<h2> </h2><div class="background"></div>';
+    return section;
+  },
+
+  populateItem: function(el, i) {
+    debug('populate item', el);
+    var title = el.firstChild;
+    var body = title.nextSibling;
+    var record = this.getRecordAt(i);
+
+    title.firstChild.data = record.name;
+    body.firstChild.data = record.metadata.artist;
+  },
+
+  populateItemDetail: function(el, i) {
+    var img = el.querySelector('img');
+    img.src = makeImage(i);
+    img.onload = function() {
+      setTimeout(function() {
+        img.classList.add('loaded');
+      });
```

Comment:
maybe some leftover from before I figured out I needed to wait for the load, will try without tomorrow :)